packages/mgt-components/src/components/mgt-people/mgt-people.ts

```diff
@@ -407,6 +428,17 @@ export class MgtPeople extends MgtTemplatedComponent {
           this.people = await findGroupMembers(graph, null, this.groupId, this.showMax, PersonType.person);
         } else if (this.userIds) {
           this.people = await getUsersForUserIds(graph, this.userIds);
+          if (this._fallbackDetails) {
```

Comment:
Awesome. Could we do this for peopleQueries as well? (re #1346)

Reply:
Had to add some handling for the query since it wasn't returning null... Let me know if we should handle this differently.

e942c1e

Reply:
This is great, I think you are fixing part of the bug in #1346 with this.